flow/error_definitions.h

```diff
@@ -206,6 +203,17 @@ ERROR( key_not_found, 2400, "Expected key is missing")
 ERROR( json_malformed, 2401, "JSON string was malformed")
 ERROR( json_eof_expected, 2402, "JSON string did not terminate where expected")
 
+// 2500 - disk snapshot based backup errors
+ERROR( snap_disable_tlog_pop_failed,  2500, "Snapshot error")
```

Comment:
I think 'Snapshot error' is too generic a description

Reply:
I tried to translate this error into a way that is consumable to the end-user and felt it was very hard. So, made this a generic error. Since the fdbcli prints both the error code and the error message, I was thinking of writing a detailed error documentation associated with the error code.

Reply:
It might be good to tie the error message to the disk snapshot feature, since snapshot is also overloaded for both snapshot reads and snapshots taken by the other backup tool.

Reply:
does appending it with disk_snap_* will help?

Reply:
I think I'm ok with the names of the errors, but for the two unqualified "Snapshot error" descriptions, it might be nice to have it say something like "Disk snapshot error". Also, is there anything distinct about these two errors to the user?

Reply:
error ids are different and hope to explain the difference between them in the disk snapshot documentation.